web-components/src/components/form/CreateSellOrderForm.tsx

```diff
@@ -85,7 +87,10 @@ const CreateSellOrderForm: React.FC<FormProps> = ({
     <Formik
       initialValues={initialValues}
       validate={validateHandler}
-      onSubmit={onSubmit}
+      onSubmit={async values => {
+        track('sell2');
+        await onSubmit(values);
```

Comment:
Why do we need to wait here if we're not doing anything after?

Reply:
I think I just misunderstood something. I'm used to python async/await where everything in general needs to be awaited (i guess the async model is different and I'm not used to it). Will remove.

Reply:
But actually, an await is used in other locations as well. Is it definitely something we need to change?

ProfileModal.tsx:74
IndividualModal.tsx:71
OrganizationModal.tsx:82

Reply:
In these cases you mentioned, we're doing something after the `await` so there's a reason to block code execution. Otherwise, if we're not doing anything after (like the submission from above) we can definitely remove them.